pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -311,6 +311,8 @@ self: super: ({
   # https://github.com/NixOS/nixpkgs/issues/149692
   Agda = removeConfigureFlag "-foptimise-heavily" super.Agda;
 
+  heystone = addBuildTool super.heystone pkgs.fixDarwinDylibNames;
```

Comment:
```suggestion
  heystone = addBuildTool pkgs.fixDarwinDylibNames super.heystone;
```

Reply:
Well that was silly of me.  I guess it explains why the build succeeded.